README.md

```diff
@@ -15,83 +15,94 @@ _**Micro —** Async ES6 HTTP microservices_
 * **Simple**. Oriented for single purpose modules (function).
 * **Explicit**. No middleware. Modules declare all dependencies.
 * **Standard**. Just HTTP!
-* **Lightweight**. The package is small and the `async` transpilation fast and transparent
+* **Lightweight**. The package is small and the `async` transpilation is fast and transparent
 
-## Example
+## How to use
 
-The following example `sleep.js` will wait before responding (without blocking!)
+Install it:
 
-```js
-const {send} = require('micro')
-const sleep = require('then-sleep')
-
-module.exports = async function (req, res) {
-  await sleep(500)
-  send(res, 200, 'Ready!')
-}
+```
+npm install micro --save
```

Comment:
Command line indicator (dollar sign) missing. Also please put the flag before the package name!